platform/android/scripts/validate-license.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+
+from subprocess import call
+from subprocess import Popen, PIPE
+import sys
+
+## Run license generation
+call('cd ../../ && make android-license', shell=True)
+
+## Git diff changes
+p = Popen(['git', 'diff', '--name-only', 'LICENSE.md'], stdin=PIPE, stdout=PIPE, stderr=PIPE)
+output, err = p.communicate(b"input data that is passed to subprocess' stdin")
+if "platform/android/LICENSE.md" in output:
+   raise ValueError("""An error ocurred while validating the license generation. 
```

Comment:
Above `if-check` should return a boolean and the exception with this message should be thrown from a gradle script, otherwise, the exception message will be masked by gradle and we'll only get an error, without the message.

Reply:
Output shown in https://github.com/mapbox/mapbox-gl-native/pull/14177#issue-262975636 clearly shows the error.

Reply:
Sorry, at first glance I thought this is after running the python script only. That said, it doesn't print the python error for me locally, only marks the gradle task as failed. We can merge and see whether this will be the problem on CI as well whenever we run into it next time.

Reply:
This is the output I'm seeing, I feel this is good enough for what it's used for:
![image](https://user-images.githubusercontent.com/2151639/54840846-32288100-4cce-11e9-9fdd-90bf6c9fe935.png)